Tests/MessageGeneratorTests.swift

```diff
@@ -1,5 +1,255 @@
+        self.location = Location(coordinate: coordinate, city: "Bonn", country: "Germany")
+    }
+
+    override func tearDown() {
+        super.tearDown()
+    }
+
+    func testSolstice() {
+        let expected = [
+            "Have a magical winter solstice! The light will soon brighten up your days again.", // Winter solstice ☀️
+            "Sadly, today is a tiny bit shorter than yesterday. Enjoy it while it lasts!",
+            "We’ve reached the tipping point: we’ll have more sunlight every day now!",
+            "We’ve reached the tipping point: we’ll have more sunlight every day now!",
+            "We’ll have about a minute of extra light today. It’s upwards from here.",
+            "We’ve reached the tipping point: we’ll have more sunlight every day now!",
+            "About a minute of extra light. You’ll start noticing the difference soon!",
+            "About a minute of extra light. You’ll start noticing the difference soon!",
+            "Little less than a minute of extra sunlight today. It’s getting better!",
+            "We’ll have about a minute of extra light today. It’s upwards from here.",
+            "We’ll have about a minute of extra light today. It’s upwards from here.",
+            "Bring out your shorts, because today has 1 minute more sunlight.",
+            "Smile! Today has 1 minute more daylight than yesterday!",
+            "After darkness comes daylight. 1 minute more to be precise!",
+            "Today is 1 minute longer. It’s getting better and better!",
+            "1 minute more daylight today. Just let it sink in…",
+            "2 minutes extra sunshine today. Make them count!",
+            "After darkness comes daylight. 2 minutes more to be precise!",
+            "Today is 2 minutes longer than yesterday. Happy days!",
+            "Make sure to soak up that vitamin D. 2 minutes more daylight today!",
+            "2 minutes more daylight today. Just let it sink in…",
+            "Today is 2 minutes longer. It’s getting better and better!",
+            "2 minutes more daylight today. Just let it sink in…",
+            "2 minutes extra sunshine today. Make them count!",
+            "Make sure to soak up that vitamin D. 2 minutes more daylight today!",
+            "2 minutes extra sunshine today. Make them count!",
+            "2 minutes more daylight today. Just let it sink in…",
+            "Today is 2 minutes longer than yesterday. Happy days!",
+            "Bring out your shorts, because today has 2 minutes more sunlight.",
+            "Smile! Today has 3 minutes more daylight than yesterday!",
+            "Smile! Today has 3 minutes more daylight than yesterday!"
+        ]
+
+        let initialDate = self.initialDate
+        let futureDates = initialDate.next30days()
+        var actual = [String]()
+
+        // Expected string should match date list
+        XCTAssertEqual(expected.count, futureDates.count)
```

Comment:
So you expect the array of messages to have a equal  count as the number of future dates? It's more like testing the tests right? That's oke but please make the comment clearer.  // Expected strings count should match date list count